misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/ZkLeaseModule.kt

```diff
@@ -1,50 +1,15 @@
 package misk.clustering.zookeeper
 
-import com.google.common.util.concurrent.Service
-import com.google.inject.Key
-import com.google.inject.Provides
-import misk.clustering.lease.LeaseManager
-import misk.concurrent.ExecutorServiceModule
 import misk.inject.KAbstractModule
-import misk.inject.asSingleton
-import misk.inject.toKey
-import misk.tasks.RepeatedTaskQueue
-import misk.zookeeper.ZookeeperModule
-import java.time.Clock
-import java.util.concurrent.ExecutorService
-import javax.inject.Singleton
+import misk.zookeeper.FixedEnsembleProviderModule
+import misk.zookeeper.ZookeeperCommonModule
 
 /**
  * Binds a [LeaseManager] that uses Zookeeper.
  */
 class ZkLeaseModule(private val config: ZookeeperConfig) : KAbstractModule() {
```

Comment:
And, we're now using `ZkLeaseModule` to bind the common lease code plus the ensemble provider. Skim will do the same but used a different ensemble provider